docs/graph-homomorphism-as-natural-transformation.md

```diff
@@ -19,6 +23,12 @@ An example of this in haskell is Maybe.
  * The functor G(X) is Nothing.
  * The morphism between all F(X) and G(X) is Nothing (see [this issue though](https://github.com/matthewfranglen/category-set-graph/issues/1))
 
+[Paul: Let me translate this example into my language to see if I understand it properly.  The categories C and D are both *Set*, the functor F: *Set* -> *Set* is the identity (sends each object to itself and each morphism to itself), and the functor G: *Set* -> *Set* sends each set to the empty set and each function between sets to the empty function from the empty set to itself.  You propose to construct a natural transformation F -> G.
+
+However, this requires specifying, for each set X, a function from X to the empty set.  Unfortunately in standard set theory no such function can exist if X is nonempty - see the accepted answer to this question for a good explanation: https://math.stackexchange.com/questions/789123/why-is-there-no-function-with-a-nonempty-domain-and-an-empty-range (I'm not sure how this lines up with Haskell syntax).
```

Comment:
So my initial thoughts are it's fine to have a function which is `Void -> T`. You just can't call it, as you have no value with which to call it. The definition of such a function could return a constant value. Reading that link now.